File: Frontend/PI_Angular/src/app/model/jwt-dto.ts
export class JwtDto {
    token: String;
    type: String;
    nombreUsuario: String;
    authorities: String[];
   
    constructor (
        token: String,
        type: String,
        nombreUsuario: String,
        authorities: String[]){

            this.token=token;
            this.type=type;
            this.nombreUsuario=nombreUsuario;
            this.authorities=authorities;

    }
}

File: Backend/rrosales/src/main/java/com/portfolio/rrosales/Security/Dto/JwtDto.java

package com.portfolio.rrosales.Security.Dto;



public class JwtDto {
    private String token;
    
    
    public JwtDto(String token) {
        this.token = token;
      
    }

    public JwtDto() {
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
   
}
